add denormal options (ftz and daz)

jspark1105 opened the pull request:
Differential Revision: D13526340

jspark1105 commented:
@pytorchbot retest this please

yinghai reviewed (approved):
LGTM. Maybe rebase to fix the CI error?

jspark1105 committed:
add denormal options (ftz and daz)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/15423

Reviewed By: yinghai

Differential Revision: D13526340

fbshipit-source-id: 7edb7a1cb7228162417f27d7419ecdcbca04fef9
